### ISSUE TITLE
Relationship with model inside folder

### ISSUE THREAD

**Issue by aldiand** [yidas/codeigniter-model#40]
I have problem using relationship with model inside a folder.

```
    public function test()
    {
        return $this->hasOne('restapi/Test_model', 'id_test', 'id_test');
    }
```

I already fix this problem by adding 
            // Original CodeIgniter 3 model loader
```
            get_instance()->load->model($modelName);
// start by this line
            $path = explode('/', $modelName);
            if (count($path) > 1) {
                $modelName = end($path);
            }
// end line
            $model = get_instance()->$modelName;
```
at line 1198. not sure if more bug will come.

Thank you

**Comment by yidas** [comment]
Hi @aldiand,

I think you can just use model name without namespace for relationship function.

Thank you for your advice.

**Comment by aldiand** [comment]
Thanks for reply,
I got error can't find model if I just use model name, because my model is inside restapi folder.
maybe autoload may fix it, but I don't want to put it into autoload because the number of models

**Comment by yidas** [comment]
Hi @aldiand,

Do you use other libraries for autoloading at the same time? 
For example: https://github.com/yidas/codeigniter-psr4-autoload

**Comment by aldiand** [comment]
No, I just use this library without bundle

**Comment by yidas** [comment]
@aldiand,

Thank you for your advice.

The issue has been fixed in version [2.19.2](https://github.com/yidas/codeigniter-model/releases/tag/2.19.2).

Cheers

**Comment by aldiand** [comment]
You're welcome
